app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryUpdate.kt

```diff
@@ -125,6 +125,14 @@ class BloodPressureEntryUpdate(
     }
   }
 
+  private fun validateEnteredBp(model: BloodPressureEntryModel): Validation {
+    return when {
+      model.systolic.isBlank() -> Validation.ErrorSystolicEmpty
+      model.diastolic.isBlank() -> Validation.ErrorDiastolicEmpty
```

Comment:
Why do we need to extract the blank checks from the validation?

Reply:
Because `BloodPressureReading` stores the values as integers, while the model stores it as strings.

I think a long term solution would be to remove the blank validation errors and make them a part of the Mobius loop directly instead of validation errors, but I can do that later.

Reply:
Oh got it!